packages/moertel/src/mortar/mrtr_overlap_Def.hpp

```diff
@@ -1438,26 +1464,37 @@ bool MOERTEL::Overlap<IFace>::Triangulation()
       // Moertel occasionally trys to project a slave segment on a master segment which can be perpendicular. 
```

Comment:
"tries" (spelling)

Reply:
Thanks for pointing that! I read the comments but had not seen the error.